src/components/GameListItem.vue

```diff
@@ -27,7 +27,22 @@
         >
           <v-icon v-if="isRanked" class="mr-4" size="medium" icon="mdi-trophy" />
           {{ buttonText }}
+  
         </v-btn>
+        <v-btn
+          color="primary"
+          rounded
+          variant="outlined"
+          min-width="200"
+          :disabled="!allowDelete()"
```

Comment:
The name for this is a little confusing, and you're invoking it immediately. Change this to `canDeleteGame` or similar. Naming is hard. :) -- could also go the other was since we're only using it to disable the button and inline a direct check.
```suggestion
          :disabled="this.status || this.numPlayers <= 0"
```